buildscripts/packaging/Linux+BSD/org.musescore.MuseScore.appdata.xml.in

```diff
@@ -208,7 +208,7 @@
                     <li>Multiple improvements to the Plugin API</li>
                     <li>Implement changing palette and palette cell properties on fly</li>
                     <li>Current workspace is automatically saved on each action that changes the workspace</li>
-                    <li>Palettes and workspaces can be reset to the default state defined by the workspace you started customisations from</li>
+                    <li>Palettes and workspaces can be reset to the default state defined by the workspace you started customizations from</li>
```

Comment:
This file is completly and utterly outdated, the release history ends wirh 4.0.1 and has no usefull data since 3.3.4

Reply:
It's not my fault. Useless files should be removed from the repo.

Reply:
Trie, your change there looks good to me, just the rest of that file is not in a good shape